lib/utils/parseList.js

```diff
@@ -27,7 +29,14 @@ function getParseDetailList (opts) {
         lang: opts.lang,
         country: opts.country,
         cache: opts.cache
-      });
+      })
+        .then((app) => {
```

Comment:
Yes I think they should be independent, otherwise the API would be counterintuitive